src/Microsoft.TestPlatform.PlatformAbstractions/netstandard/Tracing/PlatformEqtTrace.cs

```diff
@@ -33,38 +33,32 @@ public class PlatformEqtTrace : IPlatformEqtTrace
 
     public void WriteLine(PlatformTraceLevel traceLevel, string message)
     {
-        if (!ShouldTrace(traceLevel))
-        {
-            return;
-        }
-
         var level = Enum.GetName(typeof(PlatformTraceLevel), traceLevel);
 
+# if NETSTANDARD1_3
+        System.IO.File.AppendAllText("C:\\temp\\log.txt", $"[{level}] {message}");
+#endif
         Debug.WriteLine($"[{level}] {message}");
     }
 
     public bool InitializeVerboseTrace(string customLogFile)
     {
-#if DEBUG
         // We don't have access to System.Diagnostics.Trace on netstandard1.3
         // so we write to Debug. No need to initialize for non-debug builds.
         return true;
-#else
-        return false;
-#endif
     }
 
     public bool InitializeTrace(string customLogFile, PlatformTraceLevel traceLevel)
     {
         _traceLevel = traceLevel;
 
-#if DEBUG
         // We don't have access to System.Diagnostics.Trace on netstandard1.3
         // so we write to Debug. No need to initialize for non-debug builds.
-        return true;
-#else
-        return false;
+# if NETSTANDARD1_3
+
+        System.IO.Directory.CreateDirectory("C:\\temp\\");
```

Comment:
This might not be needed if we use env var but if we keep it this should be replaced by the env var anyways.

Reply:
reverted